src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/model/OffenderManagementUnit.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.prisonregister.model
+
+import org.hibernate.annotations.GenericGenerator
+import org.hibernate.annotations.Parameter
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.OneToOne
+import javax.persistence.PrimaryKeyJoinColumn
+
+@Entity
+data class OffenderManagementUnit(
+  @OneToOne(optional = false)
```

Comment:
Spring recommends that Entities are plain Kotlin classes, not data classes, but in this case it makes sense, for now...